spec/listing_all_peeps_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+feature 'User browses the list of peeps' do
+
+  before(:each) {
+    Peep.create(:message => 'Hi')}
+
+  scenario 'when opening the home page' do
+    visit '/'
+    expect(page).to have_content('Hi')
+  end
+end
```

Comment:
This test is failing because you are creating a Peep directly in the database with a time of `nil`, as your controller handles assigning the timestamp.